### DISCUSSION TITLE
Pliki współdzielone w Azure

### DISCUSSION THREAD

**Discussion by jmadejek** [AzureCommunityPL/group#111] [Pomoc] [answered]
Firma przymierza się do likwidacji tradycyjnego serwera plików i szuka rozwiązania chmurowego (praca zdalna)
Założenia:
- przestrzeń 3TB
- pliki max ok 1GB ale pewnie może być i więcej
- pliki z programów graficznych z dość częstym dostępem do nich
- 10 użytkowników

Macie jakiś pomysł jak to sensownie ugryźć?

**Comment by smereczynski** [comment, marked as answer]
A dostaniemy procent od zysku na zleceniu? :)

**Reply by jmadejek** [reply]
To nie jest moje zlecenie tylko zapytanie o rozwiązanie. I nie mam z takich rzeczy dodatkowych pieniędzy, więc nie za bardzo jest co dzielić :/
Możesz mi podpowiedzieć w jakie rozwiązanie iść - resztę poszukam sobie sam.

**Reply by smereczynski** [reply]
Pytam, bo brzmi jak pytanie o konkretną architekturę, a nie pytanie o wsparcie po napotkaniu jakiegoś problemu. Zobacz sobie Azure Files.

**Reply by jmadejek** [reply]
OK, dzięki za kierunek, już czytam dokumentację

**Reply by mifurm** [reply]
> OK, dzięki za kierunek, już czytam dokumentację

@jmadejek - skoro to udziały to jeszcze NetApp Files jeśli potrzebujesz w przyszłości większej skali / wydajności.
